README.md

```diff
@@ -628,3 +628,27 @@ solid.web.del(url).then(
   }
 )
 ```
+
+### Managing Resource Permissions
+
+Add or modify the permissions for a particular resource or container programatically. This basically to enable applications manage the permissions of resources and containers it creates or has access to. There are 3 levels of permissions:
```

Comment:
Thanks for adding the docs!

Don't forget to hard-wrap the text lines at 80 characters.